README.md

```diff
@@ -5,59 +5,53 @@
 ![GitHub License](https://img.shields.io/github/license/charmed-hpc/hpc-libs)
 [![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
 
-A collection of charm libraries for authoring HPC charms 📖🖋️
+A collection of python libraries for authoring HPC charms 📖🖋️
```

Comment:
[nit]: I'd just say libraries. People will be able to see that the repository is primarily Python:

```suggestion
A collection of libraries for authoring HPC charms 📖🖋️
```